voice/bxml/callbacks/initiate.md

```diff
@@ -27,7 +27,36 @@ Content-Type: application/xml; charset=utf-8
+| [diversion](#diversion-properties) | (optional) Information from the most recent `Diversion` header, if any. |
+| identity			| (optional) The value of the `Identity` header from the inbound invite request. Only present if the account is configured to forward this header. |
+| [stirShaken](#stirshaken-properties) | (optional) The verification status provided by Bandwidth STIR/SHAKEN implementation. |
+
+#### Diversion properties
+
+The `Diversion` header is parsed into a JSON object. Note that the attributes vary depending on the parameters received in the SIP header. Some common attributes and values are listed below.
+
+**Note:** it is not an exhaustive list. Your application must be tolerant to unlisted attributes and values.
+
+| Property  | Description |
+|:----------|:------------|
+| origTo 	| Indicates the last telephone number that the call was diverted from. This attribute is always present.
+| reason 	| The diversion reason. Common values: `unknown`, `user-busy`, `no-answer`, `unavailable`, `unconditional`, `time-of-day`, `do-not-disturb`, `deflection`, `follow-me`, `out-of-service`, `away`.
+| screen 	| `no` if the number was provided by the user, `yes` if the number was provided by the network.
+| privacy 	| `off` or `full`.
+| counter 	| The number of diversions that have occurred.
+| limit 	| The maximum number of diversions allowed for this session.
+
+#### STIR/SHAKEN properties
+
+The Bandwidth STIR/SHAKEN implementation will verify the information provided in the inbound invite request `Identity` header. The verification status is included in the Initiate event `stirShaken` property as follows.
+
+| Property          | Description |
+|:------------------|:------------|
+| verstat | (optional) The verification status indicating whether the verification was successful or not. Possible values are `TN-Verification-Passed` or `TN-Verification-Failed`. |
```

Comment:
This is very picky so feel free to ignore. Both are possible values so I think it should be `and` instead of `or`. 

```suggestion
| verstat | (optional) The verification status indicating whether the verification was successful or not. Possible values are `TN-Verification-Passed` and `TN-Verification-Failed`. |
```